cleaners/zsh.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
99+% of Linux users do not use zsh or fish, so please submit those to https://github.com/bleachbit/cleanerml/ . Otherwise, I can accept it here and move them afterwards.

Bash is preinstalled on a typical Linux system, and here is a reference point
https://trends.google.com/trends/explore?geo=US&q=zsh,bash